### ISSUE TITLE
Applescript?

### ISSUE THREAD

**Issue by AtomicNess123** [mattcanty/zen#2]
Is this to be implemented in Applescript or from terminal? Thanks.

**Comment by mattcanty** [comment]
Personally, I run it from Terminal. Whenever I want to do some work or write, I just run:

`zen`

**Comment by AtomicNess123** [comment]
Thanks.

**Comment by AtomicNess123** [comment]
Would it be possible to run it from a key shortcut? I never use the terminal, so I would have to open it everytime. Thanks!

**Comment by mattcanty** [comment]
Are you on a Mac? If so you should be able to use Automator with a bit of searching on the internet.

**Comment by AtomicNess123** [comment]
Yes, I am on a Mac, and I spend yesterday trying to find the automator bit, but couldn't manage to make it work.

**Comment by mattcanty** [comment]
I’ve never used it having struggled with it in the past - and this morning.

I use Terminal all day during work so it’s little effort to run the command.

Perhaps there is another way, I’m not sure.
